src/app/features/workflows/components/workflow-group/workflow-group.component.ts

```diff
@@ -10,15 +10,15 @@ import { WorkflowGroupStore } from './workflow-group.store'
         <h2
           class="py-2 px-2 dark:bg-gray-700 dark:text-gray-300 bg-gray-400 text-gray-900 dark:text-gray-100 flex justify-between"
         >
-          <div>
+          <div class="flex items-center">
             <button class="text-gray-700 dark:text-gray-300 mr-2" (click)="toggleCollapse()">
-              <i class="fa fa-fw {{ !vm.isCollapsed ? 'fa-minus-circle' : 'fa-plus-circle' }}"></i>
+              <ui-icon size="md" [icon]="vm.isCollapsed ? 'plusCircle' : 'minusCircle'"></ui-icon>
```

Comment:
nitpick (non-blocking): I am super tempted to return a `collapseIcon` from the `vm`.

Reply:
nitpick: isn't `ui-icon` default `size` `md`?

Reply:
Thanks! Great points, just implemented them.